backend/src/main/java/de/kaiheinrich/projectplace/service/ProfileService.java

```diff
@@ -12,15 +20,48 @@
 @Service
 public class ProfileService {
 
+    @Value("${aws.access.key}")
+    private String accessKey;
+
+    @Value("${aws.secret.key}")
+    private String secretKey;
+
     private final ProfileMongoDb profileDb;
 
+
     @Autowired
     public ProfileService(ProfileMongoDb profileDb) {
         this.profileDb = profileDb;
     }
 
     public List<Profile> getProfiles() {
-        return profileDb.findAll();
+
+        Regions clientRegion = Regions.EU_CENTRAL_1;
+        BasicAWSCredentials awsCredentials = new BasicAWSCredentials(accessKey, secretKey);
+        String bucketName = "kais-super-bucket";
+
+        AmazonS3 s3Client = AmazonS3ClientBuilder.standard()
+                .withRegion(clientRegion).withCredentials(new AWSStaticCredentialsProvider(awsCredentials))
+                .build();
```

Comment:
I think you can put these outside of the method 🤔 they don't have to be redefined on every call